.golangci.yml

```
@@ -60,4 +60,4 @@ linters-settings:
       - "^\\s*@"
   goheader:
     template: |-
-      Copyright {{ YEAR }} PingCAP, Inc. Licensed under Apache-2.0.
+      Copyright {{ YEAR-RANGE }} PingCAP, Inc. Licensed under Apache-2.0.
```

We can bump year in a new PR.

```suggestion
      Copyright {{ YEAR }} PingCAP, Inc. Licensed under Apache-2.0.
```

---

> We can bump year in a new PR.

Should we bump all years in the goheader from 2021 to 2022?